Integration field support

jovitabruziene opened the pull request:
A temporary improvement for integration field - if fragment type is unknown and fragment is not null then return json data as Text

srenault commented:
Hey @jovitabruziene, first thanks for your contribution.
We solved the miss of integration field support in the java kit by adding a new fragment type `Raw`. Could you do the same for the csharp kit? (https://github.com/prismicio/java-kit/blob/master/src/main/java/io/prismic/Fragment.java#L1602)

srenault commented:
Hey @jovitabruziene! Thanks for your PR. I've planned to look at it in the next days.

srenault commented on src/prismic/Document.cs:
```diff
@@ -95,6 +95,9 @@ public class Document: WithFragments
 			}
 			var type = (string)json["type"];
+			if (type == null)
+                		return fragments;
```
Could you tell me when we encounter this case?

jovitabruziene replied:
Seems that this is not encountered, at least I cannot reproduce the situation. So I removed these lines of code in the new commit

srenault commented:
Your PR seems nice. Could you add tests?

jovitabruziene commented:
@srenault sorry for plenty of commits, it's becayse travis check failed at first, so I had to remove everything and add again. I've removed unnecessary (type == null) test in document.cs and added a json file for raw data type test and a test method.

srenault commented on src/prismic.tests/FragmentsTests.cs:
```diff
@@ -110,6 +110,21 @@ public void ShouldParseOEmbed()
+		[Test()]
+		public void ShouldAccessRaw()
+		{
+		    var directory = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```
We already have an helper function for this: https://github.com/prismicio/csharp-kit/blob/master/src/prismic.tests/Fixtures.cs#L10. Could you use it?

srenault commented:
Great it's ok for me.

jovitabruziene commented:
@srenault great! I've updated prismicio to version 1.4.4, but still cannot access method Document.GetRaw(), do you have any idea, why?

srenault commented:
Weird. Are you sure you're getting the right version. Because NuGet dashboard tells me `0 downloads`
<img width="799" alt="screen shot 2018-05-17 at 12 44 43" src="https://user-images.githubusercontent.com/1011289/40172864-39a9681e-59d0-11e8-8241-6fa795401c72.png">

jovitabruziene commented:
Yes. I'm getting error about compatibility with .net core, but this was present in 1.4.3 too.![image](https://user-images.githubusercontent.com/38945203/40174967-5f6b2c4a-59df-11e8-950a-d85022207746.png)

jovitabruziene commented:
@srenault , any news regarding this new release? I cannot access GetRaw method :(

srenault commented:
It seems you're using .NET Core which is not compatible with this library.

jovitabruziene commented:
In this file https://github.com/prismicio/csharp-kit/blob/master/src/prismic/WithFragments.cs there is a method public fragments.Raw GetRaw(String field)
And in 1.4.4 release I cannot find it
![image](https://user-images.githubusercontent.com/38945203/40426456-cbf3ff04-5ea3-11e8-8011-957a0ba60186.png)

srenault commented:
Ok I understand. Weird. I can try to republish it.

srenault commented:
@jovitabruziene I re-published the library and it seems to work fine now. Sorry for the delay.
